src/lib/fbxAnimation/mixamoVRMRigMap.ts

```
@@ -0,0 +1,103 @@
+/**
+ * A map from Mixamo rig name to VRM Humanoid bone name
+ */
+export const mixamoVRMRigMap = {
+	mixamorigHips: 'hips',
```

ESLint

[error] Replace `↹` with `··`

(prettier/prettier)